src/explore-education-statistics-admin/src/pages/release/content/components/KeyStatistics.module.scss

```diff
@@ -6,10 +6,14 @@
   }
 }
 
-.formContainer {
+.keyStatDataBlockFormContainer {
   width: 100%;
 }
 
+.keyStatTextFormContainer {
```

Comment:
As above, no need to have the `keyStat` prefix. Could just be `textFormContainer`